packages/apps/web-viewer-test/package.json

```diff
@@ -4,35 +4,35 @@
   "private": true,
   "dependencies": {
     "@bentley/icons-generic": "^1.0.13",
-    "@itwin/appui-abstract": "^3.2.0",
-    "@itwin/appui-layout-react": "^3.2.0",
-    "@itwin/appui-react": "^3.2.0",
+    "@itwin/appui-abstract": "4.0.0-dev.22",
```

Comment:
ok try this out, outside of the cospace like i had mentioned earlier
i can guarantee this will not work as is